app/src/main/java/com/android/voyageur/ui/notifications/AndroidNotificationProvider.kt

```diff
@@ -0,0 +1,29 @@
+package com.android.voyageur.ui.notifications
+
+import android.content.Context
+import androidx.core.app.NotificationCompat
+import com.android.voyageur.R
+
+open class AndroidNotificationProvider(private val context: Context) : NotificationProvider {
+  override fun showNewFriendRequestNotification(senderName: String) {
```

Comment:
I see that the code structures for showNewFriendRequestNotification and showFriendRequestAcceptedNotification are very similar - you could refactor the code by extracting the common part into a separate function! This would reduce code duplication and also make it easier to add new types of notifications in the future. For example, you could create a 
`private showNotification(titleResId: Int, messageResId: Int, userName: String)` 
method that handles the common notification creation logic :)

Reply:
I am not sure I understand you comment since this is the current implementation.